src/probe_modules/module_tcp_cisco_backdoor.c

```diff
@@ -44,16 +44,16 @@ static int synscan_init_perthread(void *buf, macaddr_t *src, macaddr_t *gw,
 	return EXIT_SUCCESS;
 }
 
-// instead of settings sequence number to be random for validation
-// let's instead set to something static so that we can easily
-// set acknowledgement number. I don't know how integer overflow
-// is going to act in this.
-// uint32_t tcp_seq = validation[0];
-// From Mandiant
-// 1. To initiate the process, a uniquely crafted TCP SYN packet is sent
-//	to port 80 of the “implanted” router. It is important to note that
-//	the difference between the sequence and acknowledgment numbers must
-//	be set to 0xC123D. Also the ACK number doesn’t need to be zero.
+	// instead of settings sequence number to be random for validation
+	// let's instead set to something static so that we can easily
+	// set acknowledgement number. I don't know how integer overflow
+	// is going to act in this.
+	// uint32_t tcp_seq = validation[0];
+	// From Mandiant
+	// 1. To initiate the process, a uniquely crafted TCP SYN packet is sent
+	//	to port 80 of the “implanted” router. It is important to note
+	// that the difference between the sequence and acknowledgment numbers
+	// must be set to 0xC123D. Also the ACK number doesn’t need to be zero.
```

Comment:
This indentation doesn't make sense.

Reply:
The entire comment, the part inside the comment, or both? Either way, I agree, I was just making clang-format happy. I'll look into how to fix/why it's doing this.

Reply:
I'd see if you make Clang Format just not edit the interior of comments at all.

Reply:
I updated so it doesn't reflow within comments. I believe it's aligning this block comment with the most recent statement (i.e. line 44 in module_tcp_cisco_backdoor.c) it does the same thing here: https://github.com/zmap/zmap/pull/501/files#diff-092345402e77b3908c6baad58a6851c4R586